### ISSUE TITLE
try diversity drive ensembles

### ISSUE THREAD

**Issue by jose-r-morera** [jose-r-morera/LITE#13]
3. Diversity-Driven Ensembles
This is the very recent work (2025/2026) that attacks the "Ensemble of 5" problem. It proposes a loss function that forces models to be different from each other, allowing you to use fewer models for the same accuracy.
Paper: Enhancing Time Series Classification with Diversity-Driven Neural Network Ensembles (IJCNN 2025)
Authors: Javidan Abdullayev, Maxime Devanne, Cyril Meyer, Ali Ismail-Fawaz, et al.
Link:[ Read on arXiv (2602.07579)](https://www.google.com/search?q=https://arxiv.org/abs/2602.07579)
Key Concept: Standard ensembles are redundant. By penalizing correlation during training, 2 models can do the work of 5.
